sklearn/utils/_mocking.py

```diff
@@ -166,9 +166,9 @@ def fit(self, X, y, **fit_params):
             Training vector, where n_samples is the number of samples and
             n_features is the number of features.
 
-        y : array-like of shape (n_samples, n_output) or (n_samples,), optional
+        y : array-like of shape (n_samples, n_output) or (n_samples,), default=None
```

Comment:
The lint tests are failing because lines are limited to a maximum of 79 characters.

```suggestion
        y : array-like of shape (n_samples, n_output) or (n_samples,), \
            default=None
```